content/docs/command-reference/remote/add.md

```diff
@@ -305,27 +305,28 @@ $ dvc remote add -d myremote hdfs://user@example.com/path
 **HDFS and WebHDFS:**
 
 Both remotes, HDFS and WebHDFS, allow using a Hadoop cluster as a remote
-repository. However, HDFS relies on `pyarrow` which in turn requires `libhdfs`,
-an interface to the Java Hadoop client, that must be installed separately.
-Meanwhile, WebHDFS has no need for this requirement as it communicates with the
-Hadoop cluster via a HTTP REST API using the Python libraries `HdfsCLI` and
-`requests`. The latter remote should be preferred by users who seek easier and
-more portable setups, at the expense of performance due to the added overhead of
-HTTP.
-
-One last note: WebHDFS does require enabling the HTTP REST API in the cluster by
-setting the configuration property `dfs.webhdfs.enabled` to `true` in
-`hdfs-site.xml`.
+repository. However, HDFS requires `libhdfs`, an interface to the Java Hadoop
+client, that must be installed separately. Meanwhile, WebHDFS has no need for
+this requirement as it communicates with the Hadoop cluster via a REST API.
```

Comment:
I'm not sure why we have this note under WebHDFS and not under regular HDFS. I'll open a separate PR so no worries here. Thanks for the updates!